lib/app/service/vendor.go

```diff
@@ -342,10 +342,25 @@ func printResourceStatus(resourceFile resources.ResourceFile, manifestPath strin
 	if len(extractedImages.Images) == 0 {
 		progressReporter.PrintSubStep("Found no images to vendor in %v %v", resourceType, relPath)
 	}
+	if len(extractedImages.UnrecognizedObjects) > 0 {
+		progressReporter.PrintSubStep(color.YellowString("Some resources in %v %v weren't recognized, check --debug output",
+			resourceType, relPath))
+	}
 	for _, o := range extractedImages.UnrecognizedObjects {
 		gvk := o.GetObjectKind().GroupVersionKind()
-		progressReporter.PrintSubStep("Will skip unrecognized object in %v %v: apiVersion=%v/%v, kind=%v",
-			resourceType, relPath, gvk.Group, gvk.Version, gvk.Kind)
+		unk, err := resources.ToUnknown(o)
+		if err != nil {
+			log.WithError(err).WithFields(log.Fields{
+				"apiVersion": fmt.Sprintf("%v/%v", gvk.Group, gvk.Version),
+				"kind":       gvk.Kind,
+			}).Error("Failed to convert object to unknown resource.")
```

Comment:
For me, logging an error should be the final step of the failing process. If it's not terminal, it is not an error anymore.